test/e2e/providers/cloudprovider.go

```diff
@@ -3,7 +3,7 @@ package providers
 import (
 	"fmt"
 
-	"github.com/openshift/api/config/v1"
+	v1 "github.com/openshift/api/config/v1"
```

Comment:
looks like v1 was auto-generated and can be removed here

Reply:
absolutely! VS Code seems to like to insert those :smile: 
I've changed the import to `config`